src/openrct2/common.h

```diff
@@ -92,10 +92,17 @@ using money64 = fixed64_1dp;
 #define FIXED_1DP(whole, fraction) FIXED_XDP(1, whole, fraction)
 #define FIXED_2DP(whole, fraction) FIXED_XDP(10, whole, fraction)
 
-// Construct a money value in the format MONEY(10,70) to represent 10.70. Fractional part must be two digits.
-#define MONEY(whole, fraction) ((whole)*10 + ((fraction) / 10))
+// User defined literal to convert money literal to money32
+constexpr money32 operator"" _GBP(long double money) noexcept
+{
+    return money * 10;
+}
+
+constexpr money32 ToGBP(long double money) noexcept
```

Comment:
This is named wrong. It should be ToMoney32FromGBP or something shorter. The output is money32 not gbp.